pkg/crc/cluster/cert-renewal.go

```diff
@@ -31,43 +29,26 @@ func waitForPendingCsrs(ocConfig oc.Config) error {
 	return errors.RetryAfter(120, waitForPendingCsr, time.Second*5)
 }
 
-func RegenerateCertificates(sshRunner *ssh.Runner, ocConfig oc.Config) error {
-	sd := systemd.NewInstanceSystemdCommander(sshRunner)
-	err := sd.Start("kubelet")
-	if err != nil {
-		logging.Debugf("Error starting kubelet service: %v", err)
-		return err
-	}
-	// FIXME: regression compared to previous API
-	startedKubelet := false
-	if startedKubelet {
-		defer sd.Stop("kubelet") //nolint:errcheck
-	}
+func RegenerateCertificates(ocConfig oc.Config) error {
 	/* 2 CSRs to approve, one right after kubelet restart, the other one a few dozen seconds after
 	approving the first one
 	- First one is requested by system:serviceaccount:openshift-machine-config-operator:node-bootstrapper
 	- Second one is requested by system:node:<node_name> */
-	err = waitForPendingCsrs(ocConfig)
-	if err != nil {
+	if err := waitForPendingCsrs(ocConfig); err != nil {
 		logging.Debugf("Error waiting for first pending (node-bootstrapper) CSR: %v", err)
 		return err
 	}
-	err = ApproveNodeCSR(ocConfig)
-	if err != nil {
+	if err := ApproveNodeCSR(ocConfig); err != nil {
 		logging.Debugf("Error approving first pending (node-bootstrapper) CSR: %v", err)
 		return err
 	}
-
-	err = waitForPendingCsrs(ocConfig)
-	if err != nil {
+	if err := waitForPendingCsrs(ocConfig); err != nil {
 		logging.Debugf("Error waiting for second pending (system:node) CSR: %v", err)
 		return err
 	}
-	err = ApproveNodeCSR(ocConfig)
-	if err != nil {
+	if err := ApproveNodeCSR(ocConfig); err != nil {
```

Comment:
Half of the changes in this commit replace `err = foo(); if err != nil {}` with `if err := foo(); err != nil {}` which I'm not necessarily opposed to, but would probably be better done in a separate commit rather than cluttering this one.